src/main/java/unicash/logic/commands/GetTotalExpenditureCommand.java

```diff
@@ -21,19 +17,11 @@
  * Calculates and returns the total expenditure of a user in a given month and (optionally) category and year.
  */
 public class GetTotalExpenditureCommand extends Command {
-    public static final String COMMAND_WORD = "get_total_expenditure";
-
-    public static final String MESSAGE_USAGE = new CommandUsage.Builder()
-            .setCommandWord(COMMAND_WORD)
-            .setDescription("Retrieves the total expenditure by month with optional filters for category and year.")
-            .addParameter(PREFIX_MONTH, "Month")
-            .addParameter(PREFIX_CATEGORY, "Category", true, false)
-            .addParameter(PREFIX_YEAR, "Year", true, false)
-            .setExample(ExampleGenerator.generate(COMMAND_WORD, PREFIX_MONTH, PREFIX_CATEGORY, PREFIX_YEAR))
-            .build()
-            .toString();
-
-    public static final String MESSAGE_SUCCESS = "Your total expenditure in %1$s %2$d was %3$.2f";
+    public static final String COMMAND_WORD = CommandType.GET_TOTAL_EXPENDITURE.getCommandWords();
+
+    public static final String MESSAGE_USAGE = CommandType.GET_TOTAL_EXPENDITURE.getMessageUsage();
+
+    public static final String MESSAGE_SUCCESS = CommandType.GET_TOTAL_EXPENDITURE.getMessageSuccess();
```

Comment:
Sorry if I missed it, but how would an alternative success message be handled for this?

For context, I'm planning to add another success message for when the user specifies a category so the search also includes a category. Would I follow the same convention as the budget features? I.e. creating a separate constant for it?

Reply:
Yes, a separate constant should be created for it, since it is an alternative workflow 👍

Reply:
Cool, thanks!